frontend/src/MovieGallery/component/Home.tsx

```diff
@@ -0,0 +1,56 @@
+import UpcomingMovieGallery from "./UpcomingMovieGallery";
+import {useEffect, useState} from "react";
+import {UpcomingMovie} from "../model/UpcomingMovie";
+import axios from "axios";
+import TopRatedMovieGallery from "./TopRatedMovieGallery";
+import {TopRatedMovie} from "../model/TopRatedMovie";
+import {PopularMovie} from "../model/PopularMovie";
+import PopularMovieGallery from "./PopularMovieGallery";
+
+export default function Home(){
+
+    const [upcomingMovies, setUpcomingMovies] = useState<UpcomingMovie[]>([])
+    const [topRatedMovies, setTopRatedMovies] = useState<TopRatedMovie[]>([])
+    const [popularMovies, setPopularMovies] = useState<PopularMovie[]>([])
+
+    useEffect(() => {
+        getUpcomingMovies()
+        getTopRatedMovies()
+        getPopularMovies()
+    }, [])
+
+    function getUpcomingMovies(){
+        axios.get("https://api.themoviedb.org/3/movie/upcoming?api_key=541894e58c6ae639dc89b9c8e73a5cf8&language=de")
```

Comment:
You should make api calls from your backend save the movies and pass them to your frontend.